### DISCUSSION TITLE
Parameter expand doesn't change

### DISCUSSION THREAD

**Discussion by antonnurapkin** [flet-dev/flet#1834] [General]
I have this structure of my main page: 

[Container(expand=1), all_widgets(expand=part),Container(expand=1)], 
where part= 5

Empty containers need for adaptive padding. When I make other window sizes, I want to change the relation of containers and widgets by changing the variable.

**Comment by antonnurapkin** [comment]
But it doesn't happened after assignment of variable

**Comment by holvi-mikael** [comment]
If I understand the question correctly: You need to update `all_widgets.expand` to the new value, and call `update()` as well. Variables are not dynamically connected whenever they are used to set some other value, and thus just changing the value of `part` will not magically update the `expand` value. This is not a feature or failure of `flet`, but just how Python and most other languages work, and I would not expect this to change any time soon.
